spec/ssr/ssr_node.mjs

```diff
@@ -0,0 +1,2 @@
+import L from '../../dist/leaflet-src.js';

```

Comment:
Any reason the import here is to `leaflet-src.js` and the Deno one to `leaflet-src.esm.js`? I imagine we'd want both of these using the ESM version.

Reply:
This was the only combination that worked.

When I try to use leaflet-src.js in deno, I get following errors:
![grafik](https://github.com/Leaflet/Leaflet/assets/19800037/e7f94e5b-84c2-427b-91ae-0979d2aacb20)

And for node esm doesn't work:
![grafik](https://github.com/Leaflet/Leaflet/assets/19800037/f9cd9108-dc8b-4d4f-bf4b-1d89399147a7)

Reply:
Makes sense, let's leave it as-is